config/global.ini.php

```diff
@@ -835,6 +835,10 @@
 ; If set to 1, when rearchiving reports in the past we do not rearchive segment data with those reports. Default is 0.
 rearchive_reports_in_past_exclude_segments = 0
 
+; If set to 1, then social and search engine definitions files will be will be synchronised using the internet.
+; When set to 0 the definitions will be loaded from the local definitions (updated with core).
+enable_referrer_definition_syncs = 1
```

Comment:
Should we mention here that this setting actually requires `enable_internet_features` to be enabled?

Reply:
Can mention it 👍